saas/api/plans.py

```diff
@@ -127,6 +127,10 @@ def post(self, request, *args, **kwargs):
 
     def get_queryset(self):
         self.queryset = self.organization.plans.all()
+        is_active = self.request.query_params.get('is_active')
```

Comment:
I tried to use django-filter instead, but after wasting an hour I couldn't get it to work -- I tried a lot of combinations and I have no idea why it doesn't work.

Reply:
Using `query_params` is good. The logic seems convoluted though. I believe `self.queryset` is a class fields used as a default in the base class implementation of `get_queryset` (i.e. it shouldn't be modified).